document_page/readme/CREDITS.rst

```diff
@@ -1,4 +1,3 @@
 The development of this module has been financially supported by:
 
-* Odoo SA <http://www.odoo.com>
-* Savoir-faire Linux <http://www.savoirfairelinux.com>
```

Comment:
You shouldn't remove credits like this AFAIK

Reply:
Thanks for your suggestion, I fixed it